context-free.md

```diff
@@ -1,22 +1,30 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+It is for image designers or graphical artists. 
 
 ## What is easy to do in this language? Why is it easy?
-
+It is really easy to make squares, circles, and triangles of various colors, shapes, sizes, and locations.
```

Comment:
One good advantage of this language is what you mentioned here. In fact, the language limits the users to just a few primitive shapes (found [here](https://github.com/MtnViewJohn/context-free/wiki/Primitive-Shapes) and paths, making it easier to learn the language.